packages/shorebird_cli/test/src/commands/release/release_command_test.dart

```diff
@@ -521,5 +521,25 @@ $exception''',
         });
       });
     });
+
+    group('when no platform argument is provided', () {
+      // https://github.com/shorebirdtech/shorebird/issues/2261
+
+      setUp(() {
+        when(() => argResults['platforms']).thenReturn(const <String>[]);
+      });
+
+      test('fails and log the correct message', () async {
+        final exitCode = await runWithOverrides(command.run);
+
+        expect(exitCode, equals(ExitCode.usage.code));
+
+        verify(
+          () => logger.err(
+            'At least one platform must be specified.',
```

Comment:
Is this all the user sees?  Will they know what to do with that or how to specify a platform?

Reply:
yeah, that is the only error message that I implemented, open to suggestions on a better message :)

Reply:
@eseidel I think they will known what to do as this is probably a often used command and they just did forgot by mistake to add the platforms.

But we could eventually make this message something like:

```
No platforms were provided, use the --platforms argument to provide one or more platforms
```

Wdyt?

Reply:
I agree with @eseidel here—we should endeavor to tell the user how to fix their problems when we can.

Reply:
That is fair! I've changed, can you take another look please?